WIP-CHANGELOG.md

```diff
@@ -1,26 +0,0 @@
-### Release notes
-
-- Support for multiple connectors for dex and better support for OIDC groups.
-- Check out the [upgrade guide](migration/v0.15.x-v0.16.x/upgrade-apps.md) for a complete set of instructions needed to upgrade.
```

Comment:
I think I discussed this in another PR. But maybe we could keep this line that references the upgrade guide. Then also add a basic upgrade guide with `ck8s init` and `ck8s apply`.
Not necessary that you add it in this PR but it would be nice :slightly_smiling_face:

Reply:
Yes, I agree. But this is for a separate PR.